### PR TITLE
Saas 14.3 calendar perf yti

### PR DESCRIPTION
**Pull request by arj-odoo**
Monitor and improve performances of the calendar module by decreasing the number of SQL requests.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by tivisse** [comment]
@robodoo r+ rebase-ff

**Comment by arj-odoo** [comment]
@robodoo retry

**Comment by arj-odoo** [comment]
@robodoo r+ rebase-ff

**Comment by tivisse** [comment]
@robodoo r+

File: addons/google_calendar/tests/test_sync_odoo2google.py [hunk 1]
```diff
@@ -56,6 +58,73 @@ def test_event_creation(self):
             'extendedProperties': {'shared': {'%s_odoo_id' % self.env.cr.dbname: event.id}}
         })
 
+    @patch_api
+    @users('__system__')
+    @warmup
+    def test_event_creation_perf(self):
+        EVENT_COUNT = 100
+        partners = self.env['res.partner'].create([
+            {'name': 'Jean-Luc %s' % (i), 'email': 'jean-luc-%s@opoo.com' % (i)} for i in range(EVENT_COUNT)])
```
Comment (Yenthe666) [inline addons/google_calendar/tests/test_sync_odoo2google.py:67]: Is this "opoo.com" on purpose? :D

**Pull request closed without merging**